app/src/main/java/ch/sdp/vibester/api/ServiceBuilder.kt

```diff
@@ -0,0 +1,17 @@
+package ch.sdp.vibester.api
+
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+object ServiceBuilder {
```

Comment:
Doc ? what is this object for ?

Reply:
i have missed the comment. I will add it in my other pr.